ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ListActionsComponent.java

```diff
@@ -33,5 +34,10 @@ public boolean validate() {
     public void selectAction(String name) {
         log.info("Searching for integration action {}", name);
         getElementContainingText(Element.NAME, name).shouldBe(visible).click();
+        if (name.equalsIgnoreCase("Mention")) {
+            //Can't use getButton as the root element is different :(
+            $$(By.tagName("button")).filter(Condition.matchText("(\\s*)" + "Done" + "(\\s*)"))
+                    .shouldHaveSize(1).first().click();
+        }
```

Comment:
Imho this recalls my problem from https://github.com/syndesisio/syndesis-qe/issues/214. Fancy to refactor the whole method getElementContainingText? Is it the same case?

Reply:
Actually this is super-specific fix, therefor I'd like to see more generic solution maybe :)